src/internal/EnhancedButton.js

```diff
@@ -79,12 +73,6 @@ class EnhancedButton extends Component {
     onKeyDown: () => {},
     onKeyUp: () => {},
     onKeyboardFocus: () => {},
-    onMouseDown: () => {},
```

Comment:
This of course. But this one `onTouchStart`? I'm asking couse I've run today with problems in my app becasue of the bug.